wishlist/wishlist.css

```diff
@@ -0,0 +1,179 @@
+@import "./topnav.css";
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css");
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300&display=swap');
+@import "./card.css";
+
+:root{
+    --btn-color: #2563eb;
+    --btn-hover-clr:#0ea5e9;
+    --wishlist-clr:#e11d48;
+}
+
+body{
+    margin: 0;
+    padding: 0;
+    height: 100vh;
+    overflow-y: hidden;
```

Comment:
Your page is not scrolling probably because of this, at 100% zoom level which is the the default the cards' bottom is not visible because of this overflow hidden.